### ISSUE TITLE
Anti Spam Plugin Windows Debug Codes in Client Messages issue.

### ISSUE THREAD

**Issue by Kingwolf123** [callofduty4x/CoD4x_Server#98]
Whenever some player types any message, he is shown with a debug message on his screen only.This issue is related to anti-spam plugin
On the windows server Console whenever a player types a message the following message is given: 

```
Plugin_GetPlayerUID: This command is in CoD4X18+ deprecated. 0 returned
Say player123: haha gg
``

**Comment by Kingwolf123** [comment]
I would regard this as a bug. Probably some debugging options need to be disabled and plugin needs to be recompiled.
The In-Game player is given with Debug and some random numbers ahead.

**Comment by D4edalus** [comment]
that plugin is quite old and i don't think anyone is interested in maintaining it. if somebody is using that plugin you are welcome to fix it. which would mean updating it to the new functions and formats for playerids.

**Comment by T-Maxxx** [comment]
Whats its purpose? Prevent players from spamming to chat? You can use `g_antilag <sec>` to allow 1 message per <sec>

**Comment by Kingwolf123** [comment]
The purpose is to stop people from spamming binds etc to chat.
Does the g_antilag  applies to 1 message per 1 second per total chatbox OR 1 message per 1 second per 1 player?

**Comment by IceNinjaman** [comment]
g_antilag turns on or off serverside latency calculations when players getting shot. Without g_antilag you have to aim sufficient meters ahead of a moving player to hit him. You have to predict where a player could go next. How many meters it are depends on you ping.

**Comment by T-Maxxx** [comment]
Whoops. My bad. `sv_floodprotect`

**Comment by D4edalus** [comment]
tl;dr 

constrains player to send less than n messages per minute. 

Plugin_GetPlayerUid is deprecated, used to identify admins
can be removed from plugin ( then admins arent excluded from spamming ) 
https://github.com/callofduty4x/CoD4x_Server/blob/master/plugins/antispam/antispam_plugin.c#L49

@Kingwolf123 please make a pull request if you are interested, we dont really see a need to support this plugin

**Comment by Kingwolf123** [comment]
Oh well, 
sv_floodprotect does the job just as fine.
